exec/apk-gen.sh

```diff
@@ -37,6 +37,28 @@ then
 		done
     	fi
     	# Create a new branch that will contain only latest apk
+
+    # Remove unwanted apk files
+    rm app-fdroid-release-unsigned.apk
+    rm app-playStore-release-unaligned.apk
+    rm susi-ai-master-app-playStore-release-unaligned.apk
+
+    # Remove the json files
+    rm fdroidDebug-output.json
+    rm fdroidRelease-output.json
+    rm playStoreDebug-output.json
+    rm playStoreRelease-output.json
+
+    # Rename files accordingly
+    mv app-fdroid-debug.apk susiai-dev-debug-fdroid.apk
+    mv app-playStore-debug.apk susiai-dev-debug.apk
+    mv susi-ai-master-app-fdroid-debug.apk susiai-master-debug-fdroid.apk
+    mv susi-ai-master-app-playStore-debug.apk susiai-master-debug.apk
+    mv susi-ai-master-app-fdroid-release-unsigned.apk susiai-master-unsigned-fdroid.apk
+    mv susi-ai-master-app-playStore-release-unsigned.apk susiai-master-unsigned.apk
+    mv app-playStore-release-unsigned.apk susiai-dev-release-unsigned.apk
+    mv susi-ai-master-app-playStore-release.apk susiai-master-release.apk
```

Comment:
Too repetitive, see other scripts how we do it

Reply:
We can do using the loop, but there is an issue. We won't be able to rename it in the way that Mario wants. He gave a list that cannot be renamed using the loop. 
I went through the script present in [phimpme-android](https://github.com/fossasia/phimpme-android/blob/development/scripts/update-apk.sh) which renames via loop(keeping the end portion completely same, which is different from the one expected here)

Reply:
What's not possible?

Reply:
In badge-magic renaming of the files are done by passing it through a for loop sentence.
<img width="448" alt="Screenshot 2019-07-02 at 7 14 28 AM" src="https://user-images.githubusercontent.com/43731599/60476443-3daa2180-9c99-11e9-99c3-e5631ea5cb5c.png">

In that case, only the initial part of the name ( like the app name) can be added. But what we want here is that the entire apk to be renamed and the words like playStore, app is removed from the name.

Reply:
The similar renaming has also been done in pslab-android, there also it is done manually because of completely custom naming.

Reply:
You can still do that. First remove playStore-debug and release and then do renaming. No issue

Reply:
okk, I will do it